### ISSUE TITLE
Add a search feature

### ISSUE THREAD

**Issue by pcowgill** [ricmoo/flatworm#2]
One option for adding search functionality is [DocSearch by Algolia](https://docsearch.algolia.com/), but that may or may not be philosophically aligned with this project's approach. Thanks!

Related: https://github.com/ethers-io/ethers.js/issues/900

**Comment by ricmoo** [comment]
Yeah, this project is designed to be simple, dependency-free (as much as possible without re-inventing too many conplicated wheels) and privacy-focused, which means I’d prefer preventing third-party access to customer data…

If there is a way to make it a plug-in though, I’d be receptive to that. The config.js can run arbitrary code, so there might be some option there for a post-processing step. :)
